storage/remote/write_test.go

```diff
@@ -391,21 +428,51 @@ func TestOTLPWriteHandler(t *testing.T) {
 	req.Header.Set("Content-Type", "application/x-protobuf")
 
 	appendable := &mockAppendable{}
+	conf := config.DefaultOTLPConfig
+	conf.TranslationStrategy = config.NoTranslation
```

Comment:
Should this be inside handleOtlp? If we're using handleOtlp for several functions, I feel like the configuration should be passed as a parameter.